### PR TITLE
DOCS-2119: Update power sensor for R2D2

### PR DESCRIPTION
**Pull request by sguequierre**
All models looking good this round! 

* Update config builder screenshots and remove attribute templates

### CODE REVIEW

File: docs/components/power-sensor/ina219.md [hunk 1]
```diff
@@ -25,32 +25,7 @@ Enter a name for your sensor and click **Create**.
...
-
-```json {class="line-numbers linkable-line-numbers"}
-{
-  "i2c_bus": 1
-}
-```
-
-{{% /tab %}}
-{{< /tabs >}}
+Edit the attributes as applicable to your power sensor, according to the table below.
```
[the remark this answers is not in the document]
Comment (npentrel) [reply docs/components/power-sensor/ina219.md:28]: ^well that comment didn't work - you'll have to manually re-add

File: docs/components/power-sensor/renogy.md [hunk 2]
```diff
@@ -24,31 +24,7 @@ Enter a name for your sensor and click **Create**.
...
-
-```json {class="line-numbers linkable-line-numbers"}
-{
-  "serial_path": "/dev/serial/by-path/usb-0:1.1:1.0"
-}
-```
-
-{{% /tab %}}
-{{< /tabs >}}
+Edit the attributes as applicable to your power sensor, according to the table below.
```
Comment (npentrel) [inline docs/components/power-sensor/renogy.md:27]: see above - please keep attributes

File: docs/components/power-sensor/ina226.md [hunk 3]
```diff
@@ -23,32 +23,7 @@ Enter a name for your sensor and click **Create**.
...
-
-```json {class="line-numbers linkable-line-numbers"}
-{
-  "i2c_bus": 1
-}
-```
-
-{{% /tab %}}
-{{< /tabs >}}
+Edit the attributes as applicable to your power sensor, according to the table below.
```
Comment (npentrel) [inline docs/components/power-sensor/ina226.md:26]: see above - please keep attributes

File: docs/components/power-sensor/ina219.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (sguequierre) [reply docs/components/power-sensor/ina219.md:28]: I removed bc I feel like this duplicates the info in the JSON template a bit and is less helpful than it was before when there wasn't a proper config builder card for the sensor, but ok re-adding

[the remark this answers is not in the document]
Comment (sguequierre) [reply docs/components/power-sensor/ina219.md:28]: Bc they don't need to copy-paste JSON in builder mode anymore

**Review by npentrel — APPROVED** [review]

**Pull request merged**